webapp/src/DOMJudgeBundle/Controller/API/JudgehostController.php

```diff
@@ -689,185 +701,91 @@ public function addJudgingRunAction(
         string $hostname,
         int $judgingId
     ) {
-        $required = [
```

Comment:
Maybe it's better to have a separate endpoint for single / multiple runs? I think it now is pretty hard to denote this in Swagger?

Reply:
Well, on the one hand we don't really need the old endpoint anymore. On the other hand, it's still nice to have it around especially for swagger.
With swagger you'll never to do a batch update, so I think it's fine to just not document it in the Swagger API. What do you think?

Reply:
OK this sounds reasonable. If at some point we want to get rid of the old style of doing this, we can fix it then.